### ISSUE TITLE
MeteringConfiguration

### ISSUE THREAD

**Issue by corymosiman12** [BuildingSync/schema#183]
Currently, the `auc:MeteringConfiguration` element is a child element of an `auc:Utility`.  If we assume that an `auc:Scenario/auc:ResourceUses/auc:ResourceUse` should be defined to represent an actual WaterResource or EnergyResource, then I believe it makes more sense to define the `auc:MeteringConfiguration` element as a child of `auc:ResourceUse`.  This way, a Utility serving potentially multiple resources (water, gas, electricity, steam, etc.) could still be metered differently for each individual resource.

Thoughts? @nllong @markborkum 

If agree, I will make formal proposal.

**Comment by markborkum** [comment]
From the Audit Template perspective, it makes sense to keep things as they are. The Audit Template "energy supply source" class is mapped to/from the `auc:Utility` element and the "metering type" attribute (of said class) is mapped to/from the `auc:MeteringConfiguration` child element. Audit Template uses the `auc:UtilityID` element to relate `auc:ResourceUse` and `auc:Utility` elements when needed.

**Comment by corymosiman12** [comment]
I see.  So the assumption in ATT is that an `auc:Utility` is created for every 'Energy Supply Source' (that's what it looks like from an XML export).  I just disagree with this implementation, since I think others would assume that a Utility can serve multiple resources.  Let's discuss next call.

**Comment by markborkum** [comment]
Correct. In Audit Template, there is a one-to-one association between "energy supply sources" (`auc:Utility`) and "energy types" (`auc:FuelTypes`). In future versions, this could be generalized to a one-to-many association.

**Comment by corymosiman12** [comment]
Cool.  Let's keep this open as discussion.  As I get documentation together for how we think these concepts should be modeled, we can discuss further.
